docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
+    you can use :class:`~astropy.table.Table` instead and then write to FITS.
+    This is less complicated than "lower-level" FITS interface:
+
+    >>> from astropy.table import Table
+    >>> t = Table([[1, 2], [4, 5], [7, 8]], names=('a', 'b', 'c'))
+    >>> t.write('table.fits', format='fits')
+
+    The equivalent codes using ``astropy.io.fits`` would look like this:
```

Comment:
Well, I'm not sure either. But it sounded just wrong.

See also: http://english.stackexchange.com/q/20455/186372